charts/datree-admission-webhook/templates/_scan-job.tpl

```diff
@@ -42,7 +42,16 @@ spec:
         - name: scan-job
           env:
             - name: DATREE_TOKEN
-              value: {{.Values.datree.token}}
+            {{- if and .Values.datree.existingSecret (ne .Values.datree.existingSecret.name "") (ne .Values.datree.existingSecret.key "") }}
+            - name: DATREE_TOKEN
```

Comment:
Have a look at line 44 in this file.
in this implementation the line `- name: DATREE_TOKEN` will be rendered twice.
I see 2 optional fixes for that:
1. remove line 44
2. remove both lines 46 and 52.


I prefer the second option (seems more readable to me)
This could apply to the changes in `charts/datree-admission-webhook/templates/deployment.yaml` file too

WDYT?

Reply:
Thats a very good point. I am somewhat surprised it worked as intended with this omission, as in I expected this would shout at me when trying to deploy the helm o.O
```
- name: DATREE_TOKEN
- name DATREE_TOKEN
  value: .... 
 ```
I guess it set the value to null in the first one then overwrote it hence it "worked"